specification/callbacks.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Providing Callbacks
+parent: The OpenAPI Specification Explained
+nav_order: 9
+---
+
+# Providing Callbacks
+
+Callbacks are a feature of OAS that provides a relationship between a given [Operation](https://spec.openapis.org/oas/v3.1.0#operation-object) and a [Path Item](https://spec.openapis.org/oas/v3.1.0#path-item-object) that can be implemented by an API consumer. A callback allows an API provider to describe this relationship at design time, indicating that they can call an API consumer at a given URL based on the definition of both the Callback and dynamic values received during a given Operation. These are described as "out-of-band" by the specification as the expectation is the API provider can call such URLs independently of the Operation in which they are defined.
```

Comment:
I'm not sure this opening paragraph quite captures the purpose of callbacks. Maybe we can say something like "Callbacks are a way to describe HTTP requests sent from server to client as a followup to an earlier API call"  or something? The point here is that the HTTP requests go the _other way_ than they usually do - and for callbacks, they only do so as a followup to an earlier API call, unlike webhooks that have out-of-band registration and the requests can be sent in response to another trigger.

Reply:
i would agree that

i would rather start with this parapgraph 

callbacks are defined as asynchronous, out-of-band requests that your service sends to another service in response to certain events.  

i would insists on the difference with webhook, that a callback is a reaction toward a first incoming call. That is the main differentiator, Webhook requires registration (that can be static or dynamic) 
there is no sponateous callback versus there can be for webhook